acos_client/v30/axapi_http.py

```diff
@@ -105,8 +105,7 @@ def request(self, method, api_url, params={}, headers=None,
         try:
             r = z.json()
         except ValueError as e:
-            # Suspect that the JSON response was empty, like in the case of a
-            # successful file import.
+            # The response is not JSON but it still succeeded.
             if z.status_code == 200:
                 return {}
```

Comment:
@dougwig Did you still want me to revisit this? [Related](https://github.com/dougwig/acos-client/pull/3/files#r23340700).